include/Atlas.h

```diff
@@ -45,6 +45,28 @@ class Pinhole;
 
 class Atlas
 {
+    friend class boost::serialization::access;
+
+    template<class Archive>
+    void serialize(Archive &ar, const unsigned int version)
+    {
+        //ar.template register_type<Pinhole>();
+        //ar.template register_type<KannalaBrandt8>();
+
+        //Save/Load all objects
+        ar & mspMaps;
+        ar & mvpBackupMaps;
```

Comment:
is this necessary?

Reply:
Now that you mention it, serializing mspMaps should not be required given that these are backed up in PreSave and rebuilt in PostLoad. I have not yet tested without that line if it works. If it's the case I'll make the change ;) Thanks !

Reply:
you are right, either mspMaps or mvpBackupMaps is ok. https://github.com/UZ-SLAMLab/ORB_SLAM3/pull/310/files#r627846281